src/Core/src/Handlers/Window/WindowHandler.cs

```diff
@@ -0,0 +1,44 @@
+using System;
+using Microsoft.Maui.Animations;
+#if __IOS__ || MACCATALYST
+using NativeView = UIKit.UIWindow;
+#elif MONOANDROID
+using NativeView = Android.App.Activity;
+#elif WINDOWS
+using NativeView = Microsoft.UI.Xaml.Window;
+#endif
+
+namespace Microsoft.Maui.Handlers
+{
+	public partial class WindowHandler : IWindowHandler
+	{
+		public static PropertyMapper<IWindow, WindowHandler> WindowMapper = new(ElementHandler.ElementMapper)
+		{
+			[nameof(IWindow.Title)] = MapTitle,
+		};
+
+		IAnimationManager? _animationManager;
+
+		public WindowHandler()
+			: base(WindowMapper)
+		{
+		}
+
+		public WindowHandler(PropertyMapper? mapper = null)
+			: base(mapper ?? WindowMapper)
+		{
+		}
+
+		public IAnimationManager AnimationManager =>
+			_animationManager ??= this.GetRequiredService<IAnimationManager>();
+
+#if !NETSTANDARD
+		protected override NativeView CreateNativeElement() =>
+#if __ANDROID__
+			MauiContext?.Context as NativeView ?? throw new InvalidOperationException($"MauiContext did not have a valid window.");
+#else
+			MauiContext?.Window ?? throw new InvalidOperationException($"MauiContext did not have a valid window.");
+#endif
+#endif
```

Comment:
The trick here is to just return the window from the context. The code flows the same as with a view, except that this is not a "create" but more a "already created, reuse".